README.md

```diff
@@ -115,19 +71,65 @@ driver = GraphDatabase.driver(URI, auth=AUTH)
 
 # Upsert the vector
 vector = [random() for _ in range(DIMENSION)]
-insert_query = (
-    "MERGE (n:Document {id: $id})"
-    "WITH n "
-    "CALL db.create.setNodeVectorProperty(n, 'vectorProperty', $vector)"
-    "RETURN n"
-)
+insert_query = """
+    MERGE (n:Document {id: $id})
+    WITH n
+    CALL db.create.setNodeVectorProperty(n, 'vectorProperty', $vector)
+    RETURN n
+"""
 parameters = {
     "id": 0,
     "vector": vector,
 }
 driver.execute_query(insert_query, parameters)
 ```
 
+### Performing a similarity search
```

Comment:
This order is more logical, in the sense that users first create the index, then populate it and finally use it. The question is: will users find quickly what they are looking for? In the sense that creating/populating the index is maybe not their main concern, if they want to use Neo4j in a RAG context, maybe they should see first this example, WDYT? 
I'm talking specifically about the README, it can be different in the documentation.

Reply:
I think the README is mainly to show at a quick glance what is possible with the package. Though I see your point I think it will be more confusing if the developer tries to run the examples and find that they have to run code below so I'd prefer to keep it in this order, at least until we get some complaints